src/components/Layout.jsx

```diff
@@ -54,8 +90,31 @@ const Layout = ({ children }) => {
               alt="Turku.py - programming community for women and nonbinaries"
             />
           </Link>
+          <a className="skip-link" href="#main">
+            {getTranslation(langCode, "skip-link")}
+          </a>
+          <nav>
+            <button
+              aria-label="Change content to English"
+              onClick={() => setLangCode("en")}
+            >
+              English
+            </button>
+            <button
+              aria-label="Change content to Finnish"
+              onClick={() => setLangCode("fi")}
+            >
+              Suomi
+            </button>
+            <button
+              aria-label="Change content to Swedish"
+              onClick={() => setLangCode("sv")}
+            >
+              Svenska
+            </button>
```

Comment:
I see the why you did this, and in some cases this is the perfect way to accomplish the task of having buttons for language change. However, in our case the site is constructed in a way, that when a language changes, the url changes, so the finnish site for example is in `/fi` and swedish in `/sv`. So what we need for this, is links as they redirect to the other sites. Also, we don't have the translations ready yet, so it would be a little bit confusing to have these buttons before they actually lead to somewhere. So I'm sorry, I have to ask you to remove them :/